src/parser/expression.js

```diff
@@ -679,7 +674,9 @@ export default class ExpressionParser extends LValParser {
         return id;
 
       case tt._do:
-        if (this.hasPlugin("doExpressions")) {
+        // TODO
+        if (true) {
```

Comment:
Is this `if` necessary at all? The `_do` context can only be legally entered if the plugin is enabled to begin with.